### PR TITLE
Update version to v0.5

### PR DESCRIPTION
**Pull request by astrofrog-conda-forge**

### CODE REVIEW

**Comment by astrofrog-conda-forge** [comment]
@conda-forge-admin, please rerender

**Comment by astrofrog-conda-forge** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,15 +17,14 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }}
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]: I think this is not necessary. Is it a pure python package?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,15 +17,14 @@ build:
...
+  host:
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]: I think you can delete the 
```

  build:
    - {{ compiler('c') }}
```
and replace this ``host`` for just ``build``

File: recipe/meta.yaml [hunk 3]
```diff
@@ -17,15 +17,14 @@ build:
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:17]: and in the script section, please use ``pip`` (also add pip as ``build`` dependency please)
change to 

```
python -m pip install --no-deps --ignore-installed .
```

File: recipe/meta.yaml (hunk 1 above)
Comment (astrofrog-conda-forge) [reply recipe/meta.yaml]: No, there is a C extension in the Python package

File: recipe/meta.yaml [hunk 4]
```diff
@@ -13,19 +13,16 @@ source:
...
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
     - setuptools
-    - toolchain
-    - numpy 1.10.*  # [not (win and (py35 or py36))]
-    - numpy 1.11.*  # [win and (py35 or py36)]
+    - numpy
```
Comment (marcelotrevisani) [inline recipe/meta.yaml]: please add ``pip`` as ``build`` requirements
I think that will be good to go. I hope so! :)

**Comment by astrofrog-conda-forge** [comment]
@marcelotrevisani - it works, thanks for your help!

File: recipe/meta.yaml [hunk 5]
```diff
@@ -13,19 +13,17 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
```
Comment (isuruf) [inline recipe/meta.yaml:19]: Change this to,
```yaml
build:
  - {{ compiler('c') }}
host:
```

**Pull request merged**